app/views/layouts/_tracking_scripts.html.erb

```diff
@@ -78,17 +78,31 @@ twq('track','PageView');
 </script>
 <!-- End Twitter universal website tag code -->
 
+<% if request.original_fullpath.start_with? '/signals' %>
+<!-- Hotjar Tracking Code for https://www.coinfi.com/signals -->
```

Comment:
Personal preference but I actually feel it would be clearer to indent the contents of the `<% if ... %>`. It messes up the indentation in the generated html but I feel thats a better tradeoff if it means our code is more maintainable.

Reply:
Cool, I agree, will make the change, thanks!